common/src/main/java/io/pravega/common/auth/ZKTLSUtils.java

```diff
@@ -25,4 +28,23 @@ public static void unsetSecureZKClientProperties() {
         System.clearProperty("zookeeper.ssl.trustStore.password");
     }
 
+    public static void setSecureZKServerProperties(String keyStore, String keyStorePasswordPath, String trustStore,
```

Comment:
how's setting these security properties relevant to this PR?

Reply:
I just re-organized the setting of these properties, given that I needed to add one more security property for Zookeeper 3.5.5 (`zookeeper.authProvider.x509`). With this, I also wrote tests to validate these settings.